DoTests.bat

```diff
@@ -0,0 +1,2 @@
+py -2 -m clarity_ext.automated_tests.test_cases.dilute_tests.hamilton_tests2
```

Comment:
This script we would probably change to using a python test runner, such as nosetests, and create both .sh and .bat for executing it.